CMakeLists.txt

```diff
@@ -744,6 +744,14 @@ if (Gettext_FOUND)
     list(APPEND devilutionx_lang_targets "${_lang_target}")
     list(APPEND devilutionx_lang_files "${_gmo_file}")
 
+    if(APPLE)
+      set_source_files_properties("${_gmo_file}" PROPERTIES
+        MACOSX_PACKAGE_LOCATION Resources/assets

```

Comment:
```suggestion
        MACOSX_PACKAGE_LOCATION Resources
```

Reply:
That would result in Resources/en.gmo instead of Resources/assets/en.gmo